e2e-tests/spring-rest-opensearch/src/main/java/com/opensearch/age/OpenSearchAgeRest.java

```diff
@@ -0,0 +1,69 @@
+package com.opensearch.age;
+
+import java.io.IOException;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(path = "/age")
+public class OpenSearchAgeRest {
+    @Autowired
+    private AgeRepository ageRepo;
+
+    @PostMapping("age")
+    public void post() throws IOException {
+        Age s = new Age(18);
+        ageRepo.index(s);
+    }
+
+    @GetMapping("{age}")
+    public List<Age> findByAge(@PathVariable("age") Integer age) throws IOException {
+        return ageRepo.findByAge(age);
+    }
+
+//    @GetMapping("gte/{gte}")
```

Comment:
is this code no longer needed? or to be put back in future? either remove it, or add a TODO comment if you ll need it in future

Reply:
Will be put back when testing other types of queries (eg: Range). Will add TODO comment.